openedx/core/djangoapps/programs/tests/test_utils.py

```diff
@@ -886,6 +899,18 @@ def test_course_pricing(self):
         self.assertEqual(data['full_program_price'], program_full_price)
         self.assertEqual(data['avg_price_per_course'], program_full_price / self.number_of_courses)
 
+    def test_course_pricing_when_all_course_runs_have_no_seats(self):
```

Comment:
Looks to me that there is only one course being created and used. `test_course_pricing_when_a_course_run_has_no_seats` ?